tmux.conf

```diff
@@ -27,6 +27,12 @@ set -g status-fg '#aaaaaa'
 set -g status-left ''
 set -g status-right ''
 
+# prefix + - split window horizontally
+bind-key - split-window -v  -c '#{pane_current_path}'
+
+# prefix + \\ split window vertically
```

Comment:
I think your last push might have removed the edit, this comment doesn't match the keymapping as it has the escaping in it.

Reply:
Yes, I have updated now, and also include note about escaping requirement.
Thanks!